src/sentry/static/sentry/app/views/eventsV2/results.tsx

```diff
@@ -310,6 +312,21 @@ class Results extends React.Component<Props, State> {
     return url;
   };
 
+  handleIncompatibleQuery = (incompatibleQuery: React.ReactNode) => {
+    const {organization} = this.props;
+    trackAnalyticsEvent({
+      eventKey: 'discover_v2.create_alert_incompatible',
+      eventName: 'Discoverv2: Creating an alert from discover was incompatible',
+      organization_id: parseInt(organization.id, 10),
+    });
+
+    this.setState({incompatibleQuery});
+  };
+
+  handleIncompatibleAlertClose = () => {
+    this.setState({incompatibleQuery: null});
+  };
```

Comment:
I would just inline this into the component call tbh

Reply:
inlined!